apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -35,7 +35,7 @@ const i18n = require('../locale');
 
 import inputLibraryJson from '@cdo/static/dance/ai/ai-inputs.json';
 
-import aiBotBorder from '@cdo/static/dance/ai/bot/ai-bot-border.png';
+import aiBotBorder from '@cdo/static/dance/ai/bot/ai-bot-mini-border.png';
```

Comment:
I attempted to use the .svg version of image, and the image couldn't be found so used the .png version here instead. Just curious why we're not able to use svgs here.

Reply:
Two things of note - one is that the file isn't in this directory, so I'm assuming you were importing from the `lib/blockly/media` directory where the file actually exists. If it _should_ be here, it just needs to be added.

Secondly, we've got some extra type checking on our imports that I'm not clear on. But in this case it's refusing to load this statement:

```
import aiBotBorder from '@cdo/static/dance/ai/bot/ai-bot-mini-border.svg';
```

However, _this_ statement is perfectly fine.
```
const aiBotBorder = require('@cdo/static/dance/ai/bot/ai-bot-mini-border.svg');
```

(though note that I did copy the svg file to the same directory locally).

I don't know enough about our guts to know if this is an acceptable fix, but I have seen these `require` statements in other places where the corresponding `import` wouldn't work.

Reply:
> Two things of note - one is that the file isn't in this directory, so I'm assuming you were importing from the `lib/blockly/media` directory where the file actually exists. If it _should_ be here, it just needs to be added.
Yes, I originally had the file in this directory.
https://github.com/code-dot-org/code-dot-org/pull/55051/commits/0662620418dade8ac7c4bfcb809b71021cf7ef2a

Reply:
If the image is in the lib/blockly/media directory (which it looks like it is), then it should be able to be loaded - if the `import` version isn't working but the `require` is, then this is just a matter of adding typing information of `.svg` imports (I had to do the same for PNG imports recently), here: https://github.com/code-dot-org/code-dot-org/blob/a2bd6e382e270467843b417fab1576f7292a95dd/apps/src/Globals.d.ts#L25

Re `import` vs `require`: so general guidelines are that we should always try to use `import` over `require` when possible. `require` erases any typing info, and works mostly as an escape hatch that happens to work because webpack can resolve the module at runtime. In new code, especially typescript code, we're aiming to only use `import`, but we have had to use `require` in a few places for special paths (ex. locale) or importing JS modules that TS can't resolve properly for some reason.

We do have plenty of instances of `require` in old code, though I believe Seth had some work in progress to mass convert these to `import` where possible.

Reply:
Thanks! @sanchitmalhotra126 , I vaguely remember you mentioned something about using `require` vs `import`. Thoughts?

Reply:
100% should be using import wherever possible over require, so awesome that it's just a matter of adding it into the typing file.

Reply:
Thanks @sanchitmalhotra126 ! Just added the .svg typing info